tests/rt.sh

```diff
@@ -419,8 +419,9 @@ if [[ $MACHINE_ID = hera.* ]] || [[ $MACHINE_ID = orion.* ]] || [[ $MACHINE_ID =
 else
   RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-${BL_DATE}}
 fi
+RTPWD=/scratch1/NCEPDEV/stmp4/Jun.Wang/FV3_RT/REGRESSION_TEST_INTEL
 
-INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20210428}
+INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20210210}
```

Comment:
I assume you want to update this and merge the latest changes that were made to 20210428 into your input data set with a newer date?